examples/session_example_1_simple_start.json

```diff
@@ -1,19 +1,22 @@
 {
-  "country_code": "BE",
-  "party_id": "BEC",
-  "id": "101",
-  "start_date_time": "2015-06-29T22:39:09Z",
-  "kwh": 0.00,
-  "token_uid": "012345678",
-  "token_type": "RFID",
-  "auth_method": "WHITELIST",
-  "location_id": "LOC1",
-  "evse_uid": "3256",
-  "connector_id": "1",
-  "currency": "EUR",
-  "total_cost": {
-    "excl_vat": 2.50
-  },
-  "status": "PENDING",
-  "last_updated": "2015-06-29T22:39:09Z"
+    "country_code": "NL",
+    "party_id": "STK",
+    "id": "101",
+    "start_date_time": "2020-03-09T10:17:09Z",
+    "kwh": 0.0,
+    "cdr_token": {
+        "uid": "123abc",
+        "type": "RFID",
+        "contract_id": "NL-TST-123-1"
```

Comment:
This is not a valid eMI3 eMA ID, that is not a hard requirement from OCPI, but we advise to use it. Should be NL-TST-C12345678-S